docs/fuse-box/important-smart-contracts/major-deployed-contracts.mdx

```diff
@@ -5,15 +5,30 @@ title: Major Deployed Contracts
 
 ## Core
 
-* Core Consensus - Fuse: 0x3014ca10b91cb3D0AD85fEf7A3Cb95BCAc9c0f79
-* BlockReward - Fuse: 0x63D4efeD2e3dA070247bea3073BCaB896dFF6C9B
+|Core Consensus | Fuse | 0x3014ca10b91cb3D0AD85fEf7A3Cb95BCAc9c0f79 |
+|--|---|---|
+| BlockReward | Fuse | 0x63D4efeD2e3dA070247bea3073BCaB896dFF6C9B |
 
-## Tokens&#x20;
 
-* Fuse(FUSE) - Ethereum: 0x970b9bb2c0444f5e81e9d0efb84c8ccdcdcaf84d
-* &#x20;Fuse(FUSE) - BSC: 0x5857c96DaE9cF8511B08Cb07f85753C472D36Ea3&#x20;
-* Wrapped Fuse (WFUSE) - Fuse: 0x0BE9e53fd7EDaC9F859882AfdDa116645287C629&#x20;
-* Fuse Dollar(FUSD) - Fuse: 0x249BE57637D8B013Ad64785404b24aeBaE9B098B
+| |  | |
```

Comment:
@emmaodia Please remove this.
![image](https://github.com/user-attachments/assets/fa02ebea-9064-4d92-a0d9-3f2c221141e2)

Reply:
Done.